src/service/mobile_config.proto

```diff
@@ -21,10 +21,35 @@ import "reward_manifest.proto";
 // - Keypair fields are binary encoded public keys, Rust encoding example here:
 //    https://github.com/helium/helium-crypto-rs/blob/main/src/public_key.rs#L347-L354
 
+message wifi_deployment_info {
+  // antenna id
+  uint32 antenna = 1;
+  // The height of the hotspot above ground level in whole meters
+  uint32 elevation = 2;
```

Comment:
maybe add comments for each field here providing some context of each field

Reply:
I was thinking about that but nothing comes to mind except obvious: "Antenna azimuth" (etc)